src/components/MessageBox/MessageBox.spec.js

```diff
@@ -15,3 +22,26 @@ describe('MessageBox component', () => {
 		expect(okBTN).toBeInTheDocument();
 	});
 });
+
+describe('MessageBox component with type "start"', () => {
```

Comment:
Usually in a test, there is one `describe` block which refers to the component (or function) which is to be tested. In this case it would be `describe('MessageBox', ....)`. Within the describe block you have the different tests for the component, each test described by `it('should do this and that', ...)`. So in this case it would be more like:
```js
describe('MessageBox', () => {
  it('renders a message box with a delete button if it is of type "delete"', () => {
    ... your test here ...
  }

  it('renders a message box with a start button if it is of type "start"', () => {
    ... your test here ...
  }
}
```
The advantage of doing it like this is that once you run `npm run test`, for each `describe` block all its containing tests (with `it`) will be grouped and you can then better associate, which test belongs to which component.